lecture_02/assignment_01/yo-cheng-lee/assignment_01.py

```diff
@@ -0,0 +1,32 @@
+from compas.artists import Artist
+from compas.datastructures import Mesh
+from compas.geometry import Box
+from compas.geometry import Frame
+from compas.geometry import Plane
+from compas.geometry import Projection
+from compas.geometry import Point
+from compas.geometry import Vector
+
+point = Point(5.00, 5.00, 10.00)
+xaxis = Vector(1.0000, 0.0000, 0.000)
+yaxis = Vector(0.0000, 1.0000, 3.000)
+frame = Frame(point,xaxis,yaxis)
+
+box = Box(frame,5.0,5.0,5.0)
```

Comment:
Code style: add a whitespace after comma:

```suggestion
frame = Frame(point, xaxis, yaxis)

box = Box(frame, 5.0, 5.0, 5.0)
```